text/chapter6.md

```diff
@@ -47,7 +47,15 @@ instance Show Boolean where
   show false = "false"
 ```
 
-This code declares a type class instance called `showBoolean` – in PureScript, type class instances can be named to aid the readability of the generated JavaScript. We say that the `Boolean` type _belongs to the `Show` type class_.
+This code declares a type class instance; we say that the `Boolean` type _belongs to the `Show` type class_.
+
+> Optionally, we can give names to type class instances to aid the readability of the generated JavaScript. For example:
```

Comment:
Not blocking, but would be informative to include a snippet of the generated JS containing `showBoolean`. Even better would be to include both versions with and without the custom name.

Reply:
Oh yeah, good idea 🙏